src/main/java/wolox/training/models/Users.java

```diff
@@ -49,7 +44,7 @@ public String getUsername() {
     }
 
     public void setUsername(String username) {
-        Preconditions.checkNotNull(username, "Please check the Object supplied, its null!");
+        Preconditions.checkNotNull(username, "Please check the USERNAME!!!!! supplied, its null!");
```

Comment:
This `USERNAME!!!!` sound a little bit aggressive haha

Reply:
hahahahaha I forgot to remove it